build.xml

```diff
@@ -1986,7 +2078,7 @@
     </path>
     <pathconvert property="eclipse-libs-list" refid="eclipse-project-libs-path" pathsep="${line.separator}">
         <mapper>
-            <regexpmapper from="^(.*)$$" to='&lt;classpathentry kind="lib" path="\1\" \/&gt;'/>
+            <regexpmapper from="^(.*)$$" to='  &lt;classpathentry kind="lib" path="\1" /&gt;'/>
```

Comment:
This was previously generating invalid xml in the .classpath file:

```
<classpathentry kind="lib" path="/home/tolbertam/Documents/Projects/cassandra/lib/ST4-4.0.8.jar\" \/>
```

In particular the `\/>` part is invalid but I expect `...jar\"` probably would throw things off too.  Also added two whitespaces to the beginning to make the spacing look right (not reqiuired, just prettier).